### PR TITLE
128 appeal Bare HTML for getting strings translated

### PR DESCRIPTION
**Pull request by radishmouse**
WIP - created PR so strings could be translated.

### CODE REVIEW

File: sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      <p>
+        {{_('With revolutionary code improvements, Nebula sets the stage for
+        faster delivery of new features, reinforced stability, stellar
+        quality-of-life enhancements and visual refinements.')}}
+      </p>
+    </section>
+
+    <section class="speed">
+      <header>{{_('')}}</header>
+      <p>{{_('')}}</p>
```
Comment (MelissaAutumn) [inline sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html:25]: Fun fact about empty msgid, Pontoon uses it for project details/contact info 😄  https://github.com/thunderbird/thunderbird.net-l10n/blob/master/en_CA/LC_MESSAGES/messages.po#L9

File: sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public #
+License, v. 2.0. If a copy of the MPL was not distributed with this # file, You
+can obtain one at http://mozilla.org/MPL/2.0/. -#} {% set active_page =
+"appeal-128" %} {% set page_title_text = _('Experience the future of
+Thunderbird') %} {% extends "includes/base/base.html" %} {% block page_title
+%}{{ page_title_text }}{% endblock %} {% block page_desc %}{{ _('Thunderbird is
+a free email application that’s easy to set up and customize - and it’s loaded
+with great features!') }}{% endblock %} {% block breadcrumbs %}{% endblock %} {%
+block extra_meta %}
```
Comment (MelissaAutumn) [inline sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html:9]: I think you nuked a newline or 5. If you're IDE has jinja template support then assign this file to that.

File: sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+<meta name="robots" content="noindex,nofollow" />
+{% endblock %} {% block content %}
```
Comment (MelissaAutumn) [inline sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html:11]: New line needed

File: sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      <p>
+        {{_("We've overcome major roadblocks to bring Rust into Thunderbird.
+        This paves the way for better code, shared features between desktop and
+        mobile, and faster development time. It's a game-changer for our
+        developers and for our users.")}}
+      </p>
+    </section>
+  </div>
+</section>
+{% endblock %} {% block site_footer %}
```
Comment (MelissaAutumn) [inline sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html:67]: Neeeewline

File: sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+<div class="appeal-footer-container">
+  <div class="pre-footer-cover-container">
+    {% include 'includes/components/page-separator-cover.html' %}
+  </div>
+  <div id="footer" class="container footer">
+    {% include 'includes/appeal-footer.html' %}
+  </div>
+  {% include 'includes/donation-includes.html' %}
+</div>
+{% endblock site_footer %}
```
Comment (MelissaAutumn) [inline sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html:77]: Python craves an empty new line at the end of file.

**Review by MelissaAutumn — COMMENTED** [review]
Thanks! Looking good and awaiting last minute copy edits, and new screenshots. Once those are in can you add the new copy and screenshot alt text?

It also seems you're missing the pre-footer copy.

File: sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html (hunk 2 above)
Comment (radishmouse) [reply sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html:9]: Yeah, I think prettier got a little aggressive :joy:

Comment (MelissaAutumn) [reply sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html:9]: I would exclude prettier from thunderbird-website btw. We don't have a linter setup here.

File: sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html [hunk 6]
```diff
@@ -0,0 +1,118 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% set active_page = "beta-appeal-24" %}
```
Comment (MelissaAutumn) [inline sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html:5]: Can we adjust this to whatsnew-128

**Review by MelissaAutumn — COMMENTED** [review]
Can we also rename the folder to 128.0/whatsnew/ ?

File: settings.py [hunk 7]
```diff
@@ -271,6 +271,7 @@
     'thunderbird-style': ['less/style.less'],
     # 2024 Redesign
     'base-style': ['less/base-style.less'],
+    'appeal-128': ['less/appeal-128.less'],
```
Comment (MelissaAutumn) [inline settings.py:274]:
```suggestion
    'whatsnew-128': ['less/whatsnew-128.less'],
```

**Review by MelissaAutumn — COMMENTED** [review]
One last thing, and it's good to goooo

**Pull request merged**